llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -1534,6 +1534,12 @@ def FeatureISAVersion11_5_1 : FeatureSet<
      FeatureVGPRSingleUseHintInsts,
      Feature1_5xVGPRs])>;
 
+def FeatureISAVersion11_5_2 : FeatureSet<
```

Comment:
Looks the same as 1150?

Reply:
Are they not?

Reply:
I don't know, but if they are I have a question why a new target needed?

Reply:
GFX1152 is same as GFX1150. In my internal PR, I took @jayfoad 's suggestion of adding a new target ISA version even though they are identical.

Reply:
Then I defer review to Jay.

Reply:
I don't have a good answer to this except "it's what we normally do". Other parts of the software stack (kernel drivers etc) need to distinguish gfx1150 from gfx1152, and I guess they don't want to map "gfx1152" -> "gfx1150" before invoking the compiler. Also, it will make it easier for us to implement gfx1152-specific optimizations and workarounds in future if there are any.